go/datas/commit_test.go

```diff
@@ -82,3 +84,57 @@ func TestCommitWithoutMetaField(t *testing.T) {
 	})
 	assert.False(IsCommitType(noMetaCommit.Type()))
 }
+
+func TestCommitDescendsFrom(t *testing.T) {
+	assert := assert.New(t)
+	cs := chunks.NewTestStore()
+	db := NewDatabase(cs)
+	defer cs.Close()
```

Comment:
I copied this pattern from database_test.TearDownTest. If not required, I'll remove from there as well.